src/views/AddResource.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  Layout,
+  Content,
+  Menu,
+  Header,
+  Button,
+  Input,
+  Select,
+  Search,
+  Form,
+  Affix,
+  Col,
+  Space,
+} from '../ant';
+import {Typography,  DatePicker} from 'antd'
```

Comment:
Same as the above, import Typography & DatePicker from `../ant`. You may also need to add them to the `../ant` file.